index.html

```diff
@@ -1166,36 +1223,28 @@ <h4 id="example-get-all-patient-types">Example: Get Payments Response</h4>
     { 
         cardHolderName : 'Name 1',
         authorizationCode : '323',
-        transactionStatus : transactionStatus, 
-        cardType : '3',
         description : 'description',
-        paymentProcessorSecret : 'prc_j843890',
-        creditCardType : '1',
         transactionDate : 2000-01-23T04:56:07.000+00:00,
         merchantName : 'merchantName',
-        transactionType : 'phonepayment',
-        transactionStatusId : 6,
         authorizedAmount : '132.00',
         creditCardBrand : 'visa', 
         id : 1,
-        creditCardLast4 : '1234'
+        creditCardLast4 : '1234',
```

Comment:
'id' (line 1231 & 1244) is a Guid, not an int, so maybe we want the example more like the Guid on line 1294.

Reply:
updated